libs/platform/src/lib/components/dynamic-page/dynamic-page-content/dynamic-page-content.component.html

```diff
@@ -0,0 +1,8 @@
+<ng-container *ngIf="tabLabel">
+    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+</ng-container>
+
+<ng-container *ngIf="!tabLabel">
+    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+</ng-container>
+<ng-template #contentTemplate><ng-content></ng-content></ng-template>
```

Comment:
I can't get it as well.
What is a reson to have two `<ng-container *ngTemplateOutlet="contentTemplate"></ng-container>`?
In the end it renders the same output.
Isn't it enough just to have `<ng-template #contentTemplate><ng-content></ng-content></ng-template>`?

Reply:
It looks like it should be enough:
```
<ng-container *ngIf="!tabLabel">
    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
</ng-container>

<ng-template #contentTemplate><ng-content></ng-content></ng-template>
```

Reply:
the problem is I need to show the user the same `fdp-dynamic-page-content` but render different things based on whether `tabLabel` is present or not. 
 If I were to project the tabbed content also with   `<ng-content select="fdp-dynamic-page-content"></ng-content>` way, I found that it was rendering all tab contents in a single tab.
If I were to do it in the way suggested, with only one `*ngTemplateOutlet`, it doesn't render the normal content(non-tabbed examples) at all.

I render normal content like this: 
```
<ng-container *ngIf="!isTabbed">
        <ng-content select="fdp-dynamic-page-content"></ng-content>
 </ng-container>
```
and tabbed content like this
```
<fdp-dynamic-page-tabbed-content [background]="background" [size]="size">
                            <ng-container *ngTemplateOutlet="tab.contentTemplate"></ng-container>
</fdp-dynamic-page-tabbed-content>
``` 
within `fd-tabs`. I found this to be the only way to render both types of content properly.  
The reason why `cdkScrollable` was also added in the examples and extended in the class was because now that we were now using `fdp-dynamic-page-tabbed-content` to project tabbed content, it was not detecting cdkscrollable from the DynamicPageContent class itself.

Please suggest if there is a cleaner approach to have both tabbed and normal content project and listen for scroll from the same component. I tried many combinations and this was the only thing that worked for me.
Also, as explained to Deepak above, this seems to be a valid way of projecting when multiple `ng-content`s need to be used. (https://stackoverflow.com/questions/44699469/how-to-use-multiple-ng-content-in-the-same-component-in-angular-2 explains more about this usage.)

Reply:
Can this change now that the `cdkScrollable` no longer needs to be assigned to the content? In other words can we simplify this?